adaptdl/adaptdl/torch/__init__.py

```diff
@@ -37,6 +39,15 @@
 LOG.setLevel(logging.INFO)
 
 
+def version_check(version, lib):
+    if re.match("[0-9].[0-9].[0-9]", version) and \
+            version != "0.0.0":
+        return True
+    else:
+        LOG.info("adaptdl version of {} is a customer version".format(lib))
```

Comment:
Nit: I'm not sure customer is the correct word here -- maybe user or custom?

Reply:
Yeah, it is a typo, just change it to "user".